runtime/js/unix.js

```diff
@@ -188,7 +188,8 @@ function make_unix_err_args(code, syscall, path, errno) {
       errno = -9999;
     }
     // If none of the above variants, fallback to EUNKNOWNERR(int)
-    variant = BLOCK(0, errno);
+    // errno is expected to be positive
+    variant = BLOCK(0, -errno);
```

Comment:
`Math.abs` Maybe ?

Reply:
If you look at `util.getSystemErrorMap()`, the `errno` value is always negative in nodejs. See https://ant.readthedocs.io/en/latest/errors.html:
> Implementation detail: on Unix error codes are the negated errno (or -errno), while on Windows they are defined by libuv to arbitrary negative numbers. 

So we can just take the opposite.